scripts/zones/Jugner_Forest/mobs/King_Arthro.lua

```diff
@@ -25,6 +26,26 @@ function onMonsterMagicPrepare(mob, target)
 
 end;
 
+-----------------------------------
+-- onMobSpawn Action
+-----------------------------------
+
+function onMobSpawn(mob)
+
+    -- Lock all 10 Knight Crabs so they don't respawn while King Arthro is up
+    GetMobByID(KNIGHT_CRAB1):setRespawnTime(0);
```

Comment:
we can loop this in 3 lines instead of repeating 10 times

Reply:
Adding this now.